dataland-frontend/src/components/resources/frameworkDataSearch/DatasetDisplayStatusIndicator.vue

```diff
@@ -47,18 +42,19 @@ export default defineComponent({
     displayWarning(): boolean {
       return (
         this.displayedDataset?.currentlyActive === false ||
-        this.displayedDataset?.qaStatus === QAStatus.Pending ||
+        (this.displayedDataset?.qaStatus && this.displayedDataset.qaStatus !== QaStatus.Accepted) ||
         this.areMoreDatasetsViewableSimultaneously
       );
     },
     warningMessage(): string {
-      if (this.displayedDataset?.qaStatus === QAStatus.Pending) return "This dataset is currently pending review";
+      if (this.displayedDataset?.qaStatus === QaStatus.Pending) return "This dataset is currently pending review";
+      else if (this.displayedDataset?.qaStatus === QaStatus.Rejected) return "This dataset has been rejected";
       else if (this.displayedDataset?.currentlyActive === false) return "This dataset is superseded";
       else if (this.areMoreDatasetsViewableSimultaneously) return "You are only viewing a single available dataset";
       else return "ERROR";
     },
     buttonLabel(): string {
-      if (this.displayedDataset?.qaStatus === QAStatus.Pending || this.displayedDataset?.currentlyActive === false) {
+      if (this.displayedDataset?.qaStatus === QaStatus.Pending || this.displayedDataset?.currentlyActive === false) {
         return "View Active";
```

Comment:
If no active dataset exists and you're looking at pending dataset you have the "No dataset exists" message displayed at the bottom of the screen even though you're looking at a dataset:

![image](https://github.com/d-fine/Dataland/assets/35459207/654f18bd-d677-4741-bf12-09d0e75f6de9)

Reply:
Removed, to be pushed.